xml/DialogAddonInfo.xml

```diff
@@ -271,5 +271,6 @@
 				<param name="sub_label" value="$INFO[ListItem.AddonVersion,[COLOR grey],[/COLOR]]$INFO[ListItem.AddonCreator, $LOCALIZE[31071] ]" />
 			</include>
 		</control>
+		<include>TouchBackButton</include>
```

Comment:
I've already added that condition to the TouchBackButton include itself.
The buttons are only visible when touch mode is on (working correctly), but I can change it to your method if you prefer?